ros2_ws/src/warehouse_robot/config/my_controllers.yaml

```diff
@@ -73,12 +75,20 @@ diff_cont:
 
 
 
-piston_cont:
-  ros__parameters:
+# piston_cont:
```

Comment:
should we keep these or deleting them?  Comment on why we are keeping them?

Reply:
I'll add a comment, but I'd like to keep this in case we need to switch back to the old controller